### DISCUSSION TITLE
@WindowsFindBy with java-client v9

### DISCUSSION THREAD

**Discussion by Auto81** [appium/java-client#2046] [Q&A]
Having read the migration notes from v8 to v9 i see that the Windows findby have been removed, which i'm assuming will break any windows page factory annotations. Had it been marked for removal in v8? 

I'm currently testing a product on android, ios and windows. Using the page factory annotations has been fantastic to date.

Is there an extra library i need to use going forward to support windows pagefactory annotations? Was there a technical reason it's been removed? 

@mykola-mokhnach

**Comment by Auto81** [comment]
Marked as deprecated #1938 . Released v8.6.0.

Files deleted in #2036 . Released v9.0.0.

**Reply by valfirst** [reply]
Let's start from the very simple solution: does anything stop you from adding these annotations to you project?

**Reply by Auto81** [reply]
Haven't tried it yet, was just checking if it had moved or the process had changed before making something custom

Is there a particular reason it's been removed? It's been great having android, ios and windows all wrapped together by appium to date

**Reply by valfirst** [reply]
> Is there a particular reason it's been removed?

They're not used
